lib/rest_openshift.rb

```diff
@@ -5,8 +5,8 @@ module Rest
     module OpenShift
       extend Helper
 
-      def self.populate(path, base_opts, opts)
-        populate_common("/oapi/<oapi_version>", path, base_opts, opts)
+      def self.populate(path, base_opts, opts, type)
+        populate_common("/apis/#{type}.openshift.io/<api_version>", path, base_opts, opts)
```

Comment:
Using `api_version` here is not correct. In fact all these `something.openshift.io` APIs may use different API version. But since it is less broken than before, we can merge as is to get something working. We need more investigation to figure out what might be an approach that makes sense.

/lgtm

Reply:
What do you mean by `may use different API version`?  Shouldn't they all be using the same api version?

Reply:
well initial assumpion was that there will be one global API version like 1, 2, 3. Now they have multiple `something.k8s.io` and `somehting.openshift.io` APIs and each of them can have a different independent API level. Having a single parameter for this will not work.

On the other hand it can be overridden for each call so maybe will work. As long as it works now, I'm fine.